### ISSUE TITLE
[BUG] Redundant check for valid break statements in switch structures

### ISSUE THREAD

**Issue by Konafets** [typo3-ci/TYPO3SniffPool#56]
```Sniffs/ControllStructures/ValidBreakStatementsInSwitches``` checks for the multiple break statements. This is already done by ```Sniffs/ControlStructures/SwitchDeclarationSniff```.

Remove the first sniff.

Depends on #46.
